www/src/components/sidebar/item.js

```diff
@@ -14,6 +14,19 @@ const isItemActive = (activeItemParents, item) => {
 }
 
 class Item extends React.PureComponent {
+  constructor(props, context) {
+    super(props, context)
+    this.itemRef = React.createRef()
+  }
+
+  componentDidMount() {
+    if (this.itemRef.current) {
+      this.itemRef.current.scrollIntoView({
```

Comment:
In testing, this doesn't seem to work reliably - some items will scroll, while others won't. When I log the itemRef it's accurate, but the scrolling doesn't occur. If I add a short delay with `setTimeout`, it does work...perhaps there's a cleaner way to make it more consistent?

Reply:
maybe: with `useEffect` and empty array `,[])` it runs also only one time ...

Reply:
oh - it's a `class` - no hooks

Reply:
Right, this is probably an issue with the DOM node sometimes not being available. Should we convert it to a function component with hooks?